modules/logging-upgrading-loki-schema.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-upgrading.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-upgrading-loki-schema_{context}"]
+= Upgrading the LokiStack storage schema
+
+If you are using the {clo} with the {loki-op}, the {clo} 5.9 or later supports the `v13` schema version in the `LokiStack` custom resource. Upgrading to the `v13` schema version is recommended because it is the schema version to be supported going forward.
+
+.Procedure
+
+* Add the `v13` schema version in the `LokiStack` custom resource as follows:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+# ...
+spec:
+# ...
+  storage:
+    schemas:
+    # ...
+      version: v12 # <1>
+    - effectiveDate: "<yyyy>-<mm>-<future_dd>" # <2>
+      version: v13
+# ...
+----
+<1> Do not delete. Data persists in its original schema version. Keep the previous schema versions to avoid data loss.
+<2> Set a future date that has not yet started in the Coordinated Universal Time (UTC) time zone.
++
+[TIP]
+====
+To edit the `LokiStack` custom resource, you can run the `oc edit` command:
```

Comment:
I won't hold up the merge over this, but instead of doing this tip, I'd recommend just making this the first step in the procedure.

Reply:
The reason I didn't add it as the first step is because some users prefer to use the OpenShift web console, so this tip is not applicable to all readers.

Reply:
But let me rewrite it and move the tip up.